### PR TITLE
handle DID consistently

### PR DESCRIPTION
**Pull request by aabdulwahed**
- [x] Consistent handling of The DID registry in update method.

### CODE REVIEW

File: contracts/registry/DIDRegistryLibrary.sol [hunk 1]
```diff
@@ -38,14 +38,16 @@ library DIDRegistryLibrary {
         address owner = _self.didRegisters[_did].owner;
         if (owner == address(0x0)) owner = msg.sender;
```
Comment (diminator) [inline contracts/registry/DIDRegistryLibrary.sol]:
```
if (owner == address(0)) {
    owner = msg.sender;
    _self.didRegisterIds.push(_did);
}
```

Comment (aabdulwahed) [reply contracts/registry/DIDRegistryLibrary.sol]: I did that, but this mean that owner will be allowed to register only one DID.
please check this commit: https://github.com/oceanprotocol/keeper-contracts/pull/397/commits/ee4ad2f2340fdc2ea6cfbf2eb72e7b178599a2de#diff-7f0da993b63bb42586a850d1286e3c85R42

Comment (diminator) [reply contracts/registry/DIDRegistryLibrary.sol]: no, she won't she will only push the DID once
`_self.didRegisters[_did].owner == address(0)`

Comment (diminator) [reply contracts/registry/DIDRegistryLibrary.sol]: maybe also rename `owner` to `didOwner`

**Pull request merged**